components/freertos/port/port_common.c

```diff
@@ -81,7 +81,7 @@ void esp_startup_start_app_common(void)
 
 	portBASE_TYPE res = xTaskCreatePinnedToCore(&main_task, "main",
 												ESP_TASK_MAIN_STACK, NULL,
-												ESP_TASK_MAIN_PRIO, NULL, 0);
+                                                ESP_TASK_MAIN_PRIO, NULL, ESP_TASK_MAIN_CORE);
```

Comment:
minor issue, freertos code indents using tabs (this is the upstream project indentation style), and this line has been re-indented with spaces here